snakeviz/ipymagic.py

```diff
@@ -36,3 +52,16 @@ def snakeviz_magic(line, cell=None):
 def load_ipython_extension(ipython):
     ipython.register_magic_function(snakeviz_magic, magic_kind='line_cell',
                                     magic_name='snakeviz')
+
+
+def _check_ipynb():
```

Comment:
Could you say a little more about what this does? Does it verify that the frontend is the notebook?

Reply:
Yes, It checks the current runtime is under jupyter notebook instead of ipython REPL